sqldelight-compiler/integration-tests/src/test/kotlin/com/example/testmodule/TestDatabaseImpl.kt

```diff
@@ -0,0 +1,308 @@
+package com.example.testmodule
+
+import com.example.GroupQueries
+import com.example.Player
+import com.example.PlayerQueries
+import com.example.SelectNull
+import com.example.Team
+import com.example.TeamQueries
+import com.example.TestDatabase
+import com.squareup.sqldelight.Query
+import com.squareup.sqldelight.TransacterImpl
+import com.squareup.sqldelight.core.integration.Shoots
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlDriver
+import java.lang.Void
+import kotlin.Any
+import kotlin.Int
+import kotlin.Long
+import kotlin.String
+import kotlin.collections.Collection
+import kotlin.collections.MutableList
+
+internal object TestDatabaseImplExposer {
+    val schema: SqlDriver.Schema = TestDatabaseImpl.Schema
+
+    fun newInstance(
+        driver: SqlDriver,
+        playerAdapter: Player.Adapter,
+        teamAdapter: Team.Adapter
+    ): TestDatabase = TestDatabaseImpl(driver, playerAdapter, teamAdapter)}
```

Comment:
Missing newline. KP bug?

Reply:
i think i just need to addStatement instead of addCode